llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -2593,15 +2587,15 @@ void LoopAccessInfo::analyzeLoop(AAResults *AA, LoopInfo *LI,
       IsReadOnlyPtr = true;
     }
 
-    // See if there is an unsafe dependency between a load to a uniform address and
-    // store to the same uniform address.
+    // See if there is an unsafe dependency between a load to a uniform address
+    // and store to the same uniform address.
+    MemoryLocation Loc = MemoryLocation::get(LD);
```

Comment:
sink to original position? Comment needs re-formatting due to being too long?

Reply:
Yes.

Reply:
Squashed to remove comment formatting change.